Makefile

```diff
@@ -31,7 +31,7 @@ SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
 .PHONY: all
-all: build
+all: manifests test kustomize-crd api-docs
```

Comment:
I think we need to discuss what exactly needs to be included in `all`. E.g. should we have `helm-docs` there as well? (not a suggestion, just a question to better understand our baseline for including the commands)

Reply:
I was thinking about the helm-docs as well. I decided to skip it due to very few changes will include changes to helm.

I think we can start with the ones that are defined now and we can change it if we feel the need. 
Sound okay?

Reply:
Sure :)